app/lib/dfe_sign_in.rb

```diff
@@ -9,6 +9,6 @@ def self.parse_auth_hash(openid_auth_hash)
   end
 
   def self.bypass?
-    Rails.env.development? && ENV['BYPASS_DFE_SIGN_IN'] == 'true'
+    ENV['HOSTING_ENVIRONMENT_NAME'] == 'development' && ENV['BYPASS_DFE_SIGN_IN'] == 'true'
```

Comment:
Could we use `HostingEnvironment.environment_name`?

Reply:
Yes

Reply:
See https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/914 which I opened on its own since I think this change is logical irrespective of whether we add review apps or not!